dvc/lock.py

```diff
@@ -15,6 +16,12 @@
 
 DEFAULT_TIMEOUT = 5
 
+ERROR_MESSAGE = (
+    "cannot perform the cmd since DVC is busy and locked. You can delete "
+    ".dvc/lock to fix it, but first please make sure there's no other dvc "
+    "process ongoing by running: {blue}`ps xaf | grep dvc`{nc}."
+).format(blue=colorama.Back.BLUE, nc=colorama.Back.RESET)
+
```

Comment:
Actually... I searched the repo and I see that `colorama.Fore` is being used already, probably better to use that than `Back`.

Reply:
@jorgeorpinel What colour in `Fore` should be used then?

Reply:
Just switch `Back` for `Fore` please. I opened #2784 to review this later 🙂

Reply:
I see the coloring was removed after all?

Reply:
the whole section with an exact command to run was removed. I think that was too way too much to include that.